From 806d87327f4abd684722ddca232da12deb90eb66 Mon Sep 17 00:00:00 2001
From: Wen He <wen.he_1@nxp.com>
Date: Fri, 29 Nov 2019 14:52:07 +0800
Subject: [PATCH 690/741] dt/bindings: clk: Add optional field vco-frequency

commit c71920c77d1684136c56a0c157e1b250d6cf8eaa from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

Add optional field vco-frequency to work around this restriction
the user may specify its own desired VCO frequency for this PLL.

Signed-off-by: Wen He <wen.he_1@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 .../devicetree/bindings/clock/fsl,plldig.yaml         | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/Documentation/devicetree/bindings/clock/fsl,plldig.yaml b/Documentation/devicetree/bindings/clock/fsl,plldig.yaml
index 32274e94aafc..ee5b5c61a471 100644
--- a/Documentation/devicetree/bindings/clock/fsl,plldig.yaml
+++ b/Documentation/devicetree/bindings/clock/fsl,plldig.yaml
@@ -24,6 +24,17 @@ properties:
   '#clock-cells':
     const: 0
 
+  vco-frequency:
+     $ref: '/schemas/types.yaml#/definitions/uint32'
+     description: Optional for VCO frequency of the PLL in Hertz.
+        The VCO frequency of this PLL cannot be changed during runtime
+        only at startup. Therefore, the output frequencies are very
+        limited and might not even closely match the requested frequency.
+        To work around this restriction the user may specify its own
+        desired VCO frequency for the PLL. The frequency has to be in the
+        range of 650000000 to 1300000000.
+        If not set, the default frequency is 1188000000.
+
 required:
   - compatible
   - reg
-- 
2.17.1

